scripts/Jenkinsfile

```diff
@@ -122,50 +122,45 @@ pipeline {
             }
         }
 
-        stage("Build Docker images for notebooks") {
+        stage("Build Docker images") {
             steps {
                 script {
                     def buildArgs = """--build-arg NPM_EXTRA_INDEX_URL=https://${env.param_npm_private_url}
                                        --build-arg ODAHU_PLUGIN_VERSION=${Globals.buildVersion}
                                     """.replaceAll("\\s+"," ")
-
-                    cicdLibrary.buildDockerImage(
-                        'base-notebook',
-                        '.',
-                        'containers/jupyter-stacks/Dockerfile',
-                        "${buildArgs} --build-arg NOTEBOOK_TYPE=base"
-                    )
-                    cicdLibrary.buildDockerImage(
-                        'datascience-notebook',
-                        '.',
-                        'containers/jupyter-stacks/Dockerfile',
-                        "${buildArgs} --build-arg NOTEBOOK_TYPE=datascience"
-                    )
-                    cicdLibrary.buildDockerImage(
-                        'tensorflow-notebook',
-                        '.',
-                        'containers/jupyter-stacks/Dockerfile',
-                        "${buildArgs} --build-arg NOTEBOOK_TYPE=tensorflow"
-                    )
+                    def buildStages = [:]
+                    ['base', 'datascience', 'tensorflow'].each { type ->
```

Comment:
Extract the list to a global variable

Reply:
Done in 369bad08822ef06834a803757d1cbb83c556869c.